### PR TITLE
series exact match tests #3002

### PR DESCRIPTION
**Pull request by Sild**

### CODE REVIEW

**Comment by rmakulov** [comment]
lgtm

File: src/test/java/com/axibase/tsd/api/model/series/Series.java [hunk 1]
```diff
@@ -68,10 +68,16 @@ public void setData(List<Sample> data) {
     }
 
     public void addTag(String key, String value) {
+        if(tags == null) {
+            tags = new HashMap<>();
+        }
         tags.put(key, value);
     }
 
     public void addData(Sample sample) {
```
Comment (KutuzovGregory) [inline src/test/java/com/axibase/tsd/api/model/series/Series.java]: addSample?

**Comment by KutuzovGregory** [comment]
review completed

File: src/test/java/com/axibase/tsd/api/model/entityGroup/EntityGroup.java [hunk 2]
```diff
@@ -17,12 +17,12 @@
     Map<String, String> tags = new HashMap<>();
 
     public EntityGroup() {
-        if (null != name) {
```
Comment (KutuzovGregory) [inline src/test/java/com/axibase/tsd/api/model/entityGroup/EntityGroup.java]: change package to entitygroup

**Pull request merged**